dl1_data_handler/reader.py

```diff
@@ -295,8 +340,24 @@ def __init__(self,
                             'base_name': col_name,
                             'shape': (num_tels,) + col.shape[1:],
                             'dtype': col.dtype
-                            }
-                        )
+                        }
+                    )
+
+        # Add parameters info to description
+        # working only with mono at the moment
+        if self.mode == 'mono':
+            for col_name in self.training_parameters:
```

Comment:
With this you are appending an unprocessed_example_description for each parameter selected. But what you want is that you have one unprocessed_example_description of parameters matching the shape of self.training_parameters. Similar to how different channels (charge+ peak_time) are handled in the images unprocessed_example_description.